Mod0/TechnicalDay1.md

```diff
@@ -0,0 +1,75 @@
+2. Analysis - determining what is important about the data we have observed.
+3. Action - Doing something that logically follows the analysis.
+
+For example, when determining what to where we would:
+* Observe the weather conditions by
+    * Looking outside to see if the sun is shining.
+    * Go outside to feel the tempurature.
+* Analyze our findings
+    * It is cold out.
+    * The sun is shining.
+* Take action
+    * Wearing a coat.
+    * Leaving our umbrella at home.
+
+It is important for us to understand how we make decisions, so that we can create programs that are good at making decisions. Just like humans, computer programs need to make decisions in order to operate.  Programs need to decide: which view should a user see, what should I do with the data a user gives me.
+
+## Computer Action
+
+Different computers can perform different actions.  A common action is for a computer to give a user some information.  We call this `output`.  Output is the information that a program or computer gives to a user.
+
+Work through the tutorial linked below to learn about how C# outputs information to users.  Make sure to walk through each `Try it yourself` section.
+* [Outputting Information](https://www.w3schools.com/cs/cs_output.php)
+
+## Computer Observations
+
+Computers make observations by receiving input.  A computer _only_ knows what we input into it; it has no other view of 'the world'.  To learn more about computer observations, work through the tutorials below.  Make sure to walk through each `Try it yourself` Section.
+
+* [Variables](https://www.w3schools.com/cs/cs_variables.php) & [Variables](https://www.w3schools.com/cs/cs_variables_display.php)
+* [Input](https://www.w3schools.com/cs/cs_user_input.php)
+* [Boolean](https://www.w3schools.com/cs/cs_booleans.php)
+* [Comparisons](https://www.w3schools.com/cs/cs_operators_comparison.php)
+
+## Computer Analysis
+
+The final piece of decision making for computers, is an analysis of what to do based on the input it is given.  We need to tell computers exactly what to do with any given piece of information.  Learn more about computer analysis by working through the tutorials below.  Make sure to walk through each `Try it yourself` section.
+
+* [If Statements](https://www.w3schools.com/cs/cs_conditions.php)
+* [Else](https://www.w3schools.com/cs/cs_conditions_else.php)
+* [Else If](https://www.w3schools.com/cs/cs_conditions_elseif.php)
+
+## Putting it all together
+
+In order to create an interesting computer program, we need all three of the steps above to be working together.
+
+Open [this repl](https://replit.com/@MeganMcMahon1/MakingDecisions#main.cs) and complete the following:
+1. Fork the repl so that you have your own copy.
+2. Identify as many of the tools you learned about above as you can.  Make a [comment](https://www.w3schools.com/cs/cs_comments.php) above each line where you see something you recognize.
+3. Run the program by clicking on the green play button!
+
```

Comment:
Maybe as we're checking in with folks in the breakout rooms we can ask for "volunteers" to share part of their annotated code with the class?

Reply:
Love this!